static/service-worker-registration.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+if ('serviceWorker' in navigator) {
+  window.addEventListener('load', function() {
```

Comment:
@PanJ I see. That is a valid concern. Thanks for the information!

In this case, I prefer that the service worker registration code is inlined into our bundle. We don’t need to use a separate JavaScript file.

For example, our `MainLayout` could include `<ServiceWorker />` component that does the service worker registration on `componentDidMount`.